src/test/java/TriangleTest.java

```diff
@@ -0,0 +1,31 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+public class TriangleTest {
+
+    @Test
+    void shouldReturnTriangleIsRectangular() {
+        //given
+        double lengthA = 15;
+        double lenghtB = 17;
+        double lenghtC = 8;
+
+        //when
+        String checkIfTirangleIsRectangular = Triangle.checkIfTirangleIsRectangular(lengthA,lenghtB,lenghtC);
+
+        //then
+        Assertions.assertEquals("This Triangle is not Rectangual", checkIfTirangleIsRectangular);
```

Comment:
nazwa testu`shouldReturnTriangleIsRectangular`

w teście:
`"This Triangle is not Rectangual"`

![image](https://user-images.githubusercontent.com/17989854/106511798-08972e00-64d1-11eb-8346-a29730f5287d.png)